pkg/config/log.go

```diff
@@ -97,13 +101,12 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 		seelogLogLevel = "warn"
 	}
 
-	configTemplate := fmt.Sprintf(`<seelog minlevel="%s">`, seelogLogLevel)
-
 	formatID := "common"
 	if jsonFormat {
 		formatID = "json"
 	}
 
```

Comment:
I remove the minlevel because :
* The should-I-log-based-on-current-log_level logic is already implemented in the agent (cf. https://github.com/DataDog/datadog-agent/blob/master/pkg/util/log/log.go#L103-L109, and caller like https://github.com/DataDog/datadog-agent/blob/master/pkg/util/log/log.go#L371 that are already evaluated for each logging call)
* It was the most convenient way to enable log_level increase at runtime (and it does not come with any performance penalty because we already test if a log message is eligible for logging (or not) given its log level)